<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="LAPTOP-31ESQKD1" failures="1" tests="1" name="Default test(failed)(failed)" time="0.121" errors="0" timestamp="2024-01-05T09:33:33 IST">
  <testcase classname="testng_failedtestcase.Failedtestcase" name="TC2" time="0.02">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testng_failedtestcase.Failedtestcase.TC2(Failedtestcase.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TC2 -->
  <testcase classname="testng_failedtestcase.Failedtestcase" name="TC4" time="0.0">
    <ignored/>
  </testcase> <!-- TC4 -->
  <testcase classname="testng_failedtestcase.Failedtestcase" name="TC3" time="0.0">
    <ignored/>
  </testcase> <!-- TC3 -->
  <testcase classname="testng_failedtestcase.Failedtestcase" name="TC1" time="0.0">
    <ignored/>
  </testcase> <!-- TC1 -->
</testsuite> <!-- Default test(failed)(failed) -->
